app/routes/public.js

```diff
@@ -8,7 +8,7 @@ export default Route.extend({
 
   model(params) {
     return this.store.findRecord('event', params.event_id, {
-      include: 'social-links,event-copyright,speakers-call,tax'
+      include: 'social-links,event-copyright,speakers-call,tax,owner'
```

Comment:
Why including when you're already awaiting below. Only include things which are immediately visible in the page and defer things which can be lazy loaded

Reply:
@iamareebjamal This include states that the things which I will be using for the `public` route. If I wont define this here then the `edit-speaker` route's `eventDetails` which is the child route of `public` wont be able to fetch the owner.

Reply:
If not this, I will have to make another query in the `edit-speaker` and `new-speaker` page which henceforth will increase the loadtime.

Reply:
> I will have to make another query in the edit-speaker and new-speaker page which henceforth will increase the loadtime.

It won't. This will surely increase the load time of public route without any reason

Reply:
I will optimise it once I push changes for the organiser visibility.

Reply:
Done!